src/store/actions/singleCohort.actions.js

```diff
@@ -0,0 +1,29 @@
+import { SINGLE_COHORT } from './actions.type';
+import axios from 'axios';
+
+export const setSingleCohortData = (cohort = {}) => ({
+  type: SINGLE_COHORT.SET_DATA,
+  payload: cohort,
+  id: cohort.id
+});
+
+export const setSingleCohortLoading = () => ({
+  type: SINGLE_COHORT.SET_LOADING
+});
+
+export const setSingleCohortError = () => ({
+  type: SINGLE_COHORT.SET_ERROR
+});
+
+export const fetchSingleCohortData = cohortID => dispatch => {
+  //TODO: add error handling
```

Comment:
I believe we already have error handling in here =] at the catch(error). I think you mean that we should do something if we have an error to show to the user?